instrumentation/grpc/src/test/java/brave/grpc/ITTracingClientInterceptor.java

```diff
@@ -344,6 +347,27 @@ public void clientParserTestStreamingResponse() throws Exception {
     assertThat(span.tags()).hasSize(10);
   }
 
+  @Test public void customSampler() throws Exception {
+    closeClient(client);
+
+    RpcTracing rpcTracing = RpcTracing.newBuilder(tracing).clientSampler(RpcRuleSampler.newBuilder()
+      .putRule(methodEquals("SayHelloWithManyReplies"), Sampler.NEVER_SAMPLE)
+      .build()).build();
+
+    grpcTracing = GrpcTracing.create(rpcTracing);
+    client = newClient();
+
+    // unsampled
+    assertThat(GreeterGrpc.newBlockingStub(client).sayHelloWithManyReplies(HELLO_REQUEST))
+      .hasNext(); // Request is lazy, so you must invoke the iterator
```

Comment:
Since this isn't actually what we're testing, maybe we don't need `assertThat` and can just call `hasNext` directly.

Reply:
good point